scripts/security/credential-checker.sh

```
@@ -0,0 +1,216 @@
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+echo -e "${BLUE}🔒 Dotfiles セキュリティチェック${NC}"
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+echo "📁 対象ディレクトリ: $DOTFILES_DIR"
+echo "📊 チェック開始: $(date '+%Y-%m-%d %H:%M:%S')"
+echo ""
+
+cd "$DOTFILES_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)